@feature/layout (#1)
TingChiehLin/top-movies: README.md

## Top Movies App

> Top Movies App includes a list of top 100 movies and users are able to save movie into their favorite list

Reviewer: ```suggestion
Top Movies App includes a list of top 100 movies, where you can search and save movies as favorite
```
